src/sidebar/util/is-third-party-service.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const serviceConfig = require('../service-config');
```

Comment:
Since this is a new file I decided to use modern JavaScript

Reply:
I'm fine with this. The unwritten policy I have been following recently is to use new JS (except for still using CommonJS) for all code except for continuing to use `var` rather than `const`/`let` in existing files since that would otherwise be a very glaring inconsistency.

FWIW there are some good tools for doing bulk modernisation of JS code in straightforward cases (eg. converting var => const/let, functions to arrows etc.) and I'll propose at some point that we use them to improve consistency.

Reply:
P.S. I'm still referring to the [AirBnB JavaScript style guide](https://github.com/airbnb/javascript) for how to write JS, at least when it doesn't obviously conflict with our existing practice

Reply:
That guide is still a reasonable baseline. I wouldn't worry about following every detail to the letter though.

I did try using their ESLint config in a personal project but I found that a number of the rules were more prescriptive than I felt valuable (eg. mandating the use of string interpolation in places were string concatenation was just as readable IMO). More significantly, the yapf/gofmt-like tool [prettier](https://github.com/prettier/prettier) obviates the need for a lot of lint rules about formatting minutiae in JS.